packages/PoppyPrint-Core.package/PPFormatter.class/instance/visitBraceNode..st

```diff
@@ -10,12 +10,13 @@ visitBraceNode: aNode
 	self indent: (multiLine ifTrue: [1] ifFalse: [0]) around: [
 		multiLine ifTrue: [self newLine].
 		
-		aNode elements do: [:element | | isLast |
-			isLast := element = aNode elements last.
-			
+		aNode elements do: [:element | | isLast |				
+			isLast := element = aNode elements last.			
 			(self isEmptyLineMarker: element) ifFalse: [
-				self visitNode: element.
-				isLast ifFalse: [stream nextPut: $.]].
+				
+				self visitNode: element.				
+				isLast ifFalse: [config spaceBeforePointInArray ifTrue: [stream space].  stream nextPut: $.]].
```

Comment:
looks a bit crammed, I think you could give those two statements two lines as well :D

Reply:
done